projects/core/src/auth/user-auth/config/default-auth-config.ts

```diff
@@ -24,6 +24,10 @@ export const defaultAuthConfig: AuthConfig = {
       clearHashAfterLogin: false,
       responseType: 'code',
     },
+    customLoginPage: {
+      csrfEndpoint: '/csrf',
+      loginFormEndpoint: '/login',
```

Comment:
`query:` could you help me understand what's the exact purpose of this URL? I'm not sure if I correctly understand the purpose based on property's name.